databricks/sdk/core.py

```diff
@@ -94,11 +94,16 @@ def inner() -> Dict[str, str]:
 
 @credentials_provider('runtime', [])
 def runtime_native_auth(cfg: 'Config') -> Optional[HeaderFactory]:
+    if 'DATABRICKS_RUNTIME_VERSION' not in os.environ:
+        logging.debug("Skipping runtime authentication, because DATABRICKS_RUNTIME_VERSION is not set")
```

Comment:
```suggestion
```

This logline would appear in 75% of deployments, raising suspicions. let's remove it.

Reply:
@mgyucht removing this line to get this PR merged. We can discuss adding this back in when you are back.